sys/net/gnrc/link_layer/lorawan/include/gnrc_lorawan_internal.h

```diff
@@ -240,7 +254,7 @@ void gnrc_lorawan_generate_session_keys(const uint8_t *app_nonce,
                                         uint8_t *appskey);
 
 /**
- * @brief Set datarate for the next transmission
+ * @brief Set SF and BW for the next transmission
```

Comment:
please update this

Reply:
But its actually setting SF and BW right?

Reply:
No, this is only for certain DR (e.g DR7 doesn't set these modulation parameters).
Maybe it could be rephrased to "set phy parameters from DR" or something like that

Reply:
Okay. :+1: